rag/llm/sequence2txt_model.py

```
@@ -192,6 +234,17 @@ def __init__(self, key, model_name="16k_zh", base_url="https://asr.tencentclouda
         self.client = asr_client.AsrClient(cred, "")
         self.model_name = model_name
 
+    def check_available(self) -> tuple[bool, str]:
+        """Tencent Cloud ASR transcription expects raw bytes, not a file path."""
+        try:
+            wav_data = self._generate_test_wav()
+            text, _ = self.transcription(wav_data)
+            if text.find("**ERROR**") >= 0:
+                return False, text.replace("**ERROR**: ", "").strip()
+            return True, ""
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)